HWAQA2/Helper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HWAQA2
+{
+	public class Helper
+	{
+		public double GetValueFromConsole()
+		{
+			for (int i = 0; i <= 2; i++)
+			{
+				try
+				{
+					var entry = Console.ReadLine();
+					var enteredValue = double.Parse(entry);
```

Comment:
Could you please improve the validation method to accept decimal values with both '.' and ',' decimal separators?